apps/frontend/pages/dashboard.js

```diff
@@ -193,6 +193,13 @@ const ExpLog = ({projectinit}) => {
 						Download Results
 				</button> 
 				: ' '}
+				{project['finished'] == true && project['fileOutput'] != '' ?
```

Comment:
are you sure this is the behavior you want for undefined?
![image](https://user-images.githubusercontent.com/25965766/207161119-4b7f771a-5636-4971-9fa8-e1b72e130b95.png)

Reply:
if fileOutput is undefined then something's wrong the default for fileOutput is the empty string so it should always be  either an output location or the empty string